Update cli-docs.md to reflect how to handle reaching the end of a result set for cursor pagination (#511)
zapier/visual-builder: docs/_reference/cli-docs.md

  );

  // after fetching a page, set the returned cursor for the next page,

>   // or an empty string if this was the last one.

Reviewer: Small change to sync up with https://github.com/zapier/zapier-platform/pull/703/commits/d52e81defe144dd2b171328d94f7ffbaaa1f631d:
```suggestion
  // or an empty string if the cursor is null
```
